controller/blogy.js

```diff
@@ -0,0 +1,62 @@
+const Blog = require("../model/post");
+const Comment = require("../model/comment");
+const asyncWrapper = require('../middleware/async')
+
+
+// For blog post
+const createSinglePost = asyncWrapper( async (req, res) => {
+    const { title, content, post } = req.body;
+    const newBlog = new Blog({ title, content, post });
+    await res.json({ newBlog })
+    newBlog.save();
+  });
+  
+  // to get all blogs
+const getAllPost = asyncWrapper(async (req, res) => {
+    const blogs = await Blog.find({});
+    res.json({ blogs });
+  });
+  
+  // Get blogs by id
+const getPostById = asyncWrapper(async (req, res) => {
+    const blog = await Blog.findById(req.params.id);
+    res.json({ blog });
+  });
+  
+  // To post comments to blog
+const createCommentToPost = asyncWrapper(async (req, res) => {
+    const { comment, content } = req.body;
+    const postComment = await new Comment({ comment, content });
+    postComment.save();
+  });
+  
+  // To get all comments
+const getAllComments = asyncWrapper(async (req, res) => {
+    const comment = await Comment.find(req.Comment);
```

Comment:
1. `const comment` should be `const comments`
2. What is `req.Comment`?

Reply:
req.Comment is not needed as the method is to find. I removed already. in the new commit.